test/prow-cluster-operation/options/options.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"flag"
+	"strings"
+
+	"knative.dev/pkg/testutils/clustermanager"
+)
+
+type RequestWrapper struct {
+	Request          clustermanager.GKERequest
+	BackupRegionsStr string
+	AddonsStr        string
+	NoWait           bool
```

Comment:
Seems this flag is not being used.

Reply:
This is in prep for Async deletion, I will add the function